sdk/webpubsub/azure-messaging-webpubsub/src/test/java/com/azure/messaging/webpubsub/WebPubSubServiceClientTests.java

```diff
@@ -98,11 +99,25 @@ public void testBroadcastBytes() {
 
     @Test
     public void testSendToUserString() {
+        BinaryData message = BinaryData.fromString("Hello World!");
+
         assertResponse(client.sendToUserWithResponse("test_user",
-                BinaryData.fromString("Hello World!"),
+            message,
                 new RequestOptions().addRequestCallback(request -> request.getHeaders()
                         .set("Content-Type", "text/plain")),
                 Context.NONE), 202);
+
+        assertResponse(client.sendToUserWithResponse("test_user",
+                message, WebPubSubContentType.TEXT_PLAIN, message.getLength(),
+                null, Context.NONE),
+            202);
+
+//        ByteArrayInputStream messageStream = new ByteArrayInputStream(message.toBytes());
+//        assertResponse(client.sendToUserWithResponse("test_user",
+//                BinaryData.fromStream(messageStream),
+//                WebPubSubContentType.APPLICATION_OCTET_STREAM, message.getLength(),
+//                null, Context.NONE),
+//            202);
```

Comment:
Temporary disable due to bug https://github.com/Azure/azure-sdk-for-java/issues/25168